core/arch/aarch64/instr_create.h

```diff
@@ -334,6 +334,10 @@
   instr_create_1dst_4src((dc), OP_sub, (rd), (rn), (rm_or_imm), (sht), (sha))
 #define INSTR_CREATE_svc(dc, imm) \
   instr_create_0dst_1src((dc), OP_svc, (imm))
+#define INSTR_CREATE_adr(dc, rt, imm) \
```

Comment:
There's no doxygen header for these.  Looking above: there isn't for any of these.  Thus these will not show up in the html docs at all.

OK I see above a FIXME for adding them -- but I would suggest including them on any new code.  The original suggestion, still in place, is to automatically generate all of these (including the doxygen).